qiskit_braket_provider/providers/adapter.py

```diff
@@ -521,18 +521,15 @@ def _create_qiskit_gate(
     gate_name: str, gate_params: list[Union[float, Parameter]]
 ) -> Instruction:
     gate_instance = _GATE_NAME_TO_QISKIT_GATE.get(gate_name, None)
+    if gate_instance is None:
+        raise TypeError(f'Braket gate "{gate_name}" not supported in Qiskit')
+    gate_cls = gate_instance.__class__
     new_gate_params = []
     for param_expression, value in zip(gate_instance.params, gate_params):
+        # Assumes that each Qiskit gate has one free parameter per expression
```

Comment:
Yep, that's what I was trying to say with "per expression"; it felt weird to say "each gate's parameter expressions" because the expressions themselves are referred to as the gate's parameters.